main.go

```diff
@@ -0,0 +1,67 @@
+//go:generate go run pkg/codegen/cleanup/main.go
+//go:generate go run pkg/codegen/main.go
+
+package main
+
+import (
+	"context"
+	"flag"
+	"github.com/rancher/aks-operator/controller"
+	aksv1 "github.com/rancher/aks-operator/pkg/generated/controllers/aks.cattle.io"
+	core3 "github.com/rancher/wrangler/pkg/generated/controllers/core"
+	"github.com/rancher/wrangler-api/pkg/generated/controllers/apps"
+	"github.com/rancher/wrangler/pkg/kubeconfig"
+	"github.com/rancher/wrangler/pkg/signals"
+	"github.com/rancher/wrangler/pkg/start"
+	"github.com/sirupsen/logrus"
```

Comment:
We should split std modules from external ones.

Reply:
I agree, goland sometimes is changing it